src/studio/src/designer/backend/Languages/ini/nb.ini

```diff
@@ -394,8 +397,9 @@ enum = Gyldige verdier
 field = Felt
 fields = Felter
 go_to_type = Gå til type
-group = Gruppe
+group = allOf/anyOf/oneOf
```

Comment:
You deleted it from English. Was that intentional?

Reply:
Good catch, fixed